notebooks/rag_eval_ragas.ipynb

```diff
@@ -0,0 +1,5272 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.    os.environ["OPENAI_API_KEY"] = getpass("Enter OpenAI API key:")
</pre><p>No need to do this again, you've already completed intake of the OPENAI_API_KEY above. So either here or there, either is fine, but not both.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>